src/index.js

```diff
@@ -1 +1 @@
-import './style.css';
+import './style.css';
```

Comment:
- [ ] The project produces an error when installing node modules. kindly fix this so the project can be started using webpack, also consider including all the necessary packages in the package.json if any are messing up (because of the error I didn't have the chance to test webpack)

![Screenshot_20220803_070426](https://user-images.githubusercontent.com/25227022/182536358-37669b6a-75e2-4646-bb65-32203cb8ec42.png)

Reply:
is weird because in my computer is working, try again!

Reply:
@jdbs9514, your app is not working, kindly fix it and request a review